consensus/src/counters.rs

```diff
@@ -188,48 +161,8 @@ pub static ROUND_TIMEOUT_MS: Lazy<IntGauge> = Lazy::new(|| {
 });
 
 ////////////////////////
-// SYNCMANAGER COUNTERS
+// SYNC MANAGER COUNTERS
 ////////////////////////
-/// Count the number of times we invoked state synchronization since last restart.
-pub static STATE_SYNC_COUNT: Lazy<IntCounter> = Lazy::new(|| {
-    register_int_counter!(
-        "libra_consensus_state_sync_count",
-        "Count the number of times we invoked state synchronization since last restart."
-    )
-    .unwrap()
-});
-
-/// Count the number of block retrieval requests issued since last restart.
-pub static BLOCK_RETRIEVAL_COUNT: Lazy<IntCounter> = Lazy::new(|| {
-    register_int_counter!(
-        "libra_consensus_block_retrieval_count",
-        "Count the number of block retrieval requests issued since last restart."
-    )
-    .unwrap()
-});
-
-/// Histogram of block retrieval duration.
-pub static BLOCK_RETRIEVAL_DURATION_S: Lazy<DurationHistogram> = Lazy::new(|| {
-    DurationHistogram::new(
-        register_histogram!(
-            "libra_consensus_block_retrieval_duration_s",
-            "Histogram of block retrieval duration."
-        )
-        .unwrap(),
-    )
-});
-
-/// Histogram of state sync duration.
-pub static STATE_SYNC_DURATION_S: Lazy<DurationHistogram> = Lazy::new(|| {
```

Comment:
I might have just lost touch with the codebase, but why do you think these counters are not needed anymore?

Reply:
no no no, it's just replaced by my fancy new macro from https://github.com/libra/libra/pull/4486

it's just duplicate now, the expression would be `consensus_duration_sum{op='sync_to'}` in grafana instead